<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolEditForm</class>
 <widget class="QWidget" name="ToolEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbxToolInfo">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Tool Info</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbxToolType">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Drill</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>End Mill</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Engraver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Laser</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxAutoName">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblToolType">
        <property name="text">
         <string>Tool Type</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="teNote">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblNote">
        <property name="text">
         <string>Note</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtToolParameters" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="grbxGeometry">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Geometry</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblUnits">
           <property name="text">
            <string>Units</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="hlUnuts">
           <item>
            <widget class="QComboBox" name="cbxUnits">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>mm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>inches</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbxFeedSpeeds">
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <item>
              <property name="text">
               <string>mm/sec</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mm/min</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>cm/min</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>m/min</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="diameterLabel">
           <property name="text">
            <string>Diameter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="DoubleSpinBox" name="dsbxDiameter">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="sideAngleLabel">
           <property name="text">
            <string>Side Angle</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DoubleSpinBox" name="dsbxAngle">
           <property name="suffix">
            <string> Â°</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>180.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbxCuttingParameters">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Cutting Parameters</string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblPassDepth">
           <property name="text">
            <string>Pass or Depth</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="DoubleSpinBox" name="dsbxPassDepth">
           <property name="suffix">
            <string> mm</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblWarn">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblStepover">
             <property name="text">
              <string>Stepover</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="hlayStepover">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="DoubleSpinBox" name="dsbxStepover">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DoubleSpinBox" name="dsbxStepoverPercent">
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblCut">
           <property name="text">
            <string>One-turn Cut</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="hlayOneTurnCut">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="DoubleSpinBox" name="dsbxOneTurnCut">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DoubleSpinBox" name="dsbxOneTurnCutPercent">
             <property name="suffix">
              <string> %</string>
             </property>
             <property name="decimals">
              <number>2</number>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbxFeedSpeeds">
        <property name="toolTip">
         <string/>
        </property>
        <property name="title">
         <string>Feed Speeds</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lblSpindleSpeed">
           <property name="text">
            <string>Spindle Speed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="DoubleSpinBox" name="dsbxSpindleSpeed">
           <property name="suffix">
            <string> r.p.m.</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chbxFeedRate">
           <property name="toolTip">
            <string>If checked then automatic calculate!</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Feed Rate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="DoubleSpinBox" name="dsbxFeedRate">
           <property name="suffix">
            <string> mm/sec</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="chbxPlungeRate">
           <property name="toolTip">
            <string>If checked then automatic calculate!</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Plunge Rate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DoubleSpinBox" name="dsbxPlungeRate">
           <property name="suffix">
            <string> mm/sec</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>doublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
